src/websocket.js

```diff
@@ -21,6 +21,11 @@ class WebSocket extends EventTarget {
   constructor(url, protocols) {
     super();
 
+    this.onopenCb = null;
```

Comment:
please use underscore properties to mark them as private

Reply:
The linter is not happy about that. I can edit the rules of the linter